src/inject/GPMInject/interface/toast.js

```diff
@@ -0,0 +1,27 @@
+import { remote } from 'electron';
+
+Emitter.on('pauseAfter:show', () => {
+  const toast = document.createElement('paper-toast');
```

Comment:
From memory we have another bit of code somewhere that also makes a toast (I think in the error handler) perhaps as part of this PR (to reduce code duplication) you could extract the creation of a toast to a method somewhere.